app/lib/media.js

```diff
@@ -8,7 +8,7 @@ const surveyModel = require('../models/survey-model');
 const userModel = require('../models/user-model');
 const communicator = require('./communicator');
 
-/** @enum {typeof ResourceType[keyof typeof ResourceType]} */
+/** @enum {'0' | '1'} */
```

Comment:
I'm not sure this makes sense but seems harmless for now. JSDoc did not like the original:

> ERROR: Unable to parse a tag's type expression for source file media.js in line 12 with tag title "enum" and text "{typeof ResourceType[keyof typeof ResourceType]}": Invalid type expression "typeof ResourceType[keyof typeof ResourceType]": Expected "|" but "R" found.